CWE-94-Python_darwin/pedrohbtp_pytorch-zappa-serverless/main.py

```
@@ -0,0 +1,130 @@
+
+IS_LOCAL = False
+if __name__ == '__main__':
+    IS_LOCAL = True
+    env = 'dev'
+    # Test if the values were set to know if it is running locally or on lambda
+    json_data = open('zappa_settings.json')
+    env_vars = json.load(json_data)
+    for key, val in env_vars[env]['aws_environment_variables'].items():
+        os.environ[key] = val
+    print('Set the environ')
+    app.run(debug=True, host='0.0.0.0', port=8082)
```

Ruff (0.13.2)

Use of `debug=True` in Flask app detected

(S201)